jormungandr/src/network/convert.rs

```diff
@@ -34,3 +37,35 @@ impl TryFromNetwork<net_data::Fragment> for Fragment {
         read(&data)
     }
 }
+
+// TODO: Check if this is completly compatible
+impl TryFromNetwork<HeaderHash> for BlockId {
+    fn try_from_network(header_hash: HeaderHash) -> Result<Self, Error> {
+        BlockId::try_from(header_hash.serialize_as_vec().unwrap().as_slice())
+    }
+}
+
+impl TryFromNetwork<Vec<HeaderHash>> for BlockIds {
+    fn try_from_network(block_ids: Vec<HeaderHash>) -> Result<Self, Error> {
+        try_ids_from_iter(block_ids.iter())
+    }
+}
+
+// TODO: Check if this is completly compatible
+impl TryFromNetwork<chain_impl_mockchain::header::Header> for chain_network::data::block::Header {
```

Comment:
`Header` and `Fragment` of `chain_impl_mockchain` are actually imported above via `blockcfg`.